Neos.Flow/Classes/Mvc/Controller/SimpleActionController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Neos\Flow\Mvc\Controller;
+
+use Neos\Flow\Mvc\ActionRequest;
+use Neos\Flow\Mvc\ActionResponse;
+use Neos\Flow\Mvc\Exception\NoSuchActionException;
+use Neos\Flow\SignalSlot\Exception\InvalidSlotException;
+
+/**
+ * Provides most direct access to our request/response abstraction,
+ * the request comes directly from the dispatcher and goes directly back to it.
+ *
+ * For helpers to facilitate throws, forwards, redirects:
+ * @see SpecialResponsesSupport
+ *
+ * Views or other processing needs to be added to your controller as needed,
+ * helpers will be suggested here as they become available.
+ */
+class SimpleActionController implements ControllerInterface
```

Comment:
using `Simple*` as pattern because the initial implementation blow up too much is okayish, but could maybe be improved?

Pro `SimpleActionController`:
- its obvious when people look for action controller they are able to find this one
- psr _simple_ cache uses the same naming pattern

Con:
- indicates that somewhere in the codebase there is probably codesmell :P

But i was not able to come with an alternative naming so this is probably best ^^

Reply:
Yeah happy for any other name LOL. 

We could just call it WebController 😆

Reply:
LOL. It just hit me. The most obvious one to also solve my above comments... is just use a trait which will put in the functionality of mapping action name to method name :D that seems to be it :D horray?.

And then one implement the `ControllerInterface` _that_ is sexy?

Reply:
that would work indeed, I guess @bwaidelich will hit us over the head about traits but I like it too 😂

Reply:
Implemented here https://github.com/neos/flow-development-collection/pull/3297